client/galaxy/scripts/components/ToolsView.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div v-infinite-scroll="loadMore" infinite-scroll-disabled="busy">
+        <h2 class="mb-3" style="text-align: center">
+            <span id="tools-view">Tools View</span>
+        </h2>
+        <div v-if="!loading">
+            <b-container fluid class="mb-4">
+                <b-row class="justify-content-center">
+                    <b-col md="6">
+                        <b-form-group description="Search for strings or regular expressions">
+                            <b-input-group>
+                                <b-form-input
+                                    v-model="filterText"
+                                    placeholder="Type to Search"
+                                    @keyup.native="filter('filterByText')"
+                                    @keyup.esc.native="filterText = ''"
+                                />
+                                <b-input-group-append>
+                                    <b-btn :disabled="!filter" @click="filter = ''">Clear (esc)</b-btn>
+                                </b-input-group-append>
+                            </b-input-group>
+                        </b-form-group>
+                    </b-col>
+                </b-row>
+            </b-container>
+            <isotope
+                ref="iso"
+                :options="isoOptions"
+                :list="buffer"
+                style="margin: 0 auto;"
+                @filter="filterOption = arguments[2]"
+            >
+                <b-card v-for="(info, index) in buffer" :key="index" ref="cards" class="m-2" style="width: 23rem;">
+                    <div slot="header">
+                        <b-link :href="info['url']" target="_blank">
+                            <h4 class="tools-view-name">{{ info["name"] }}</h4>
+                        </b-link>
+                        <b-badge class="tools-view-section">{{ info["section"] }}</b-badge>
+                    </div>
+                    <p class="card-text" v-html="helpSummary(info['help']) || info['description']" />
+                    <p class="card-text">
+                        <b-btn v-b-modal="'modal-' + '-' + index">Info</b-btn>
+                        <b-modal :id="'modal-' + '-' + index" centered :title="info['name']">
+                            <b>{{ info["version"] + " / " + info["id"] }}</b>
```

Comment:
Please use info.version instead. Thank you for working on this.

Reply:
Thanks for checking! Would be done